uniref-rest/src/test/java/org/uniprot/api/uniref/repository/DataStoreTestConfig.java

```diff
@@ -38,8 +38,9 @@ public SolrClient unirefSolrClient() throws URISyntaxException {
 
     @Bean
     @Profile("offline")
-    public UniRefStoreClient unirefStoreClient() {
-        return new UniRefStoreClient(VoldemortInMemoryUniRefEntryStore.getInstance("uniref"));
+    public UniRefMemberStoreClient unirefMemberStoreClient() {
+        return new UniRefMemberStoreClient(
+                VoldemortInMemoryUniRefMemberStore.getInstance("uniref-member"), 5);
     }
```

Comment:
Optional comment, it would be good if we can read the batch size from the src/test/application.properties file.